### PR TITLE
refactor: update README example screens

### PR DESCRIPTION
**Pull request by d4v3d4vE**
- took new screenshots and updated the image to keep it up to date

SVAI-12

### CODE REVIEW

**Comment by ardasnturk** [comment]
I prefer to use `assets/splash.png` instead of the first screenshot.

File: example-screens.png
Comment (donni106) [inline example-screens.png:1]: thanks! two screens were questionable:

|1|2|
|---|---|
|image|image|

the contents are tooo dummy with weird texts, random images used for multiple places etc.
could you think of other screens showing those features? i guess the service panel screen can be found in different places with more useful content.
for the menu list you could create a new versioned static content that has all correct presentable entries, you know (e.g. no datenschutz and datenschutz2)?

**Review by ardasnturk — CHANGES_REQUESTED** [review]

File: example-screens.png
[the remark this answers is not in the document]
Comment (donni106) [reply example-screens.png:1]: good point, it looks off with the rounded edges compared to the others.

[the remark this answers is not in the document]
Comment (d4v3d4vE) [reply example-screens.png:1]: changed it to the `assets/splash.png`

[the remark this answers is not in the document]
Comment (ardasnturk) [reply example-screens.png:1]: I don't find the DEVELOPMENT text in the first picture correct. Please get the splash screen from int/development branch in gitlab.

Comment (d4v3d4vE) [reply example-screens.png:1]: couldn't find a better screen for screen 1 and for simple showcase purposes there should be no issue as long as we showcase that we have a widgets feature, what we do with that image

for the second screen I've photoshopped it based on your comment and removed all categories from "Über die App" which you didn't mention.

[the remark this answers is not in the document]
Comment (donni106) [reply example-screens.png:1]: it's ok arda, it is this file as you requested: https://github.com/smart-village-solutions/smart-village-app-app/blob/master/assets/splash.png
there is no int/dev here!

**Review by donni106 — APPROVED** [review]

**Review by ardasnturk — APPROVED** [review]

File: example-screens.png
[the remark this answers is not in the document]
Comment (donni106) [reply example-screens.png:1]: :/ the screenshot in the description differs from the one in the code...there is no "DEVELOPMENT" on it...

[the remark this answers is not in the document]
Comment (donni106) [reply example-screens.png:1]: @d4v3d4vE i removed the screenshot from the description as it is not needed. the change show exactly the image and is better with direct comparison to the old one.
and: there is no need to invest time on manually changing the PR description at any time.

[the remark this answers is not in the document]
Comment (donni106) [reply example-screens.png:1]: a discussion does not mean to directly do stuff. it can be discussed first

**Pull request merged**